docs/dev/Release.md

```diff
@@ -0,0 +1,397 @@
+        ./tools/release-scripts/checksum.sh distribution/target/apache-drill-1.17.0.tar.gz
+        ```
+    13. You should see 2 new commits on top of the branch, for example:
+        ```
+        [maven-release-plugin] prepare release drill-1.17.0
+        [maven-release-plugin] prepare for next development iteration
+        ```
+        Also, you should see a new `drill-1.17.0` tag.
+        Push the new tag to your personal github repository:
+        ```
+        git push origin drill-1.17.0
+        ```
+    14. Perform the release but do the commits into personal repo
+        (since Apache doesn't allow rollbacks on master if mistakes are made):
+        ```
+        mvn release:perform -DconnectionUrl=scm:git:git@github.com:vvysotskyi/drill.git -DskipTests -Darguments="-Dgpg.passphrase=${GPG_PASSPHRASE} -DskipTests=true -DconnectionUrl=scm:git:git@github.com:vvysotskyi/drill.git"
+        ```
+        If you want to additionally check resulting archives and jars, add `-Dmaven.deploy.skip=true` flag to avoid deploying jars to the Nexus repository:
+        ```
+        mvn release:perform -DconnectionUrl=scm:git:git@github.com:vvysotskyi/drill.git -DskipTests -Darguments="-Dgpg.passphrase=${GPG_PASSPHRASE} -DskipTests=true -DconnectionUrl=scm:git:git@github.com:vvysotskyi/drill.git -Dmaven.deploy.skip=true"
+        ```
+        After checks are performed, run this command without the flag.
+    15. Deploy the release commit:
+        ```
+        git checkout drill-1.17.0
+        mvn deploy -Papache-release -DskipTests -Dgpg.passphrase=${GPG_PASSPHRASE}
+        ```
+    16. Copy release files to a local release staging directory:
+        ```
+        cp ~/src/release/drill/target/target/checkout/apache-drill-1.17.0-src.tar.gz* ~/release/1.17.0-rc0/ && \ 
+        cp ~/src/release/drill/target/target/checkout/apache-drill-1.17.0.zip* ~/release/1.17.0-rc0/ \ 
+        cp ~/src/release/drill/target/checkout/distribution/target/apache-drill-1.17.0.tar.gz* ~/release/1.17.0-rc0/ \ 
+        ```
+    17. Check if the artifacts are signed properly:
+        ```
+        ./tools/release-scripts/checksum.sh ~/release/1.17.0-rc0/apache-drill-1.17.0-src.tar.gz
+        ./tools/release-scripts/checksum.sh ~/release/1.17.0-rc0/apache-drill-1.17.0-src.zip
+        ./tools/release-scripts/checksum.sh ~/release/1.17.0-rc0/apache-drill-1.17.0.tar.gz
+        ```
+    18. Copy release files to a directory on `home.apache.org` for voting:
+        ```
+        scp ~/release/1.17.0-rc0/* <username>@home.apache.org:~/public_html/drill/releases/1.17.0/rc0
+        ```
+
+4. ## Automated release
+    Run the following script:
+    ```
+    tools/release-scripts/release.sh
+    ```
+    The release script will push the maven artifacts to the Maven staging repo.
+5. ## Publish release candidate and vote
+    1. Go to the [Apache Maven staging repo](https://repository.apache.org/) and close the new jar release.
+        This step is done in the Maven GUI.
```

Comment:
Maybe add info link: https://central.sonatype.org/pages/releasing-the-deployment.html#locate-and-examine-your-staging-repository